charts/victoria-metrics-cluster/templates/vmselect-service.yaml

```diff
@@ -35,6 +35,12 @@ spec:
       port: {{ .Values.vmselect.service.servicePort }}
       protocol: TCP
       targetPort: {{ .Values.vmselect.service.targetPort }}
+  {{- range .Values.vmselect.service.extraPorts }}
```

Comment:
I think extraPorts must be also added to the head-less service

https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-cluster/templates/vmselect-service-headless.yaml#L18

Reply:
Done!